typings/telegram-types.d.ts

```diff
@@ -522,6 +580,10 @@ export interface MessageDice extends TT.Message {
   dice: Dice
 }
 
+export interface MessagePoll extends TT.Message {
+  poll: Poll
+}
+
 export interface NewInvoiceParams {
```

Comment:
The same to here! Unneeded shortening.

Reply:
Note that I didn't picked up this name, it was already like this. Now that you mentioned it, I will improve it while renaming other stuff, but please don't expect me to fix the whole world in a single PR.